src/display/editor/editor.js

```diff
@@ -840,6 +847,32 @@ class AnnotationEditor {
     }
   }
 
+  #setAltTextButtonState() {
+    if (this.#altTextButton && (this.#altTextDecorative || this.#altText)) {
+      this.#altTextButton.classList.add("done");
+      if (this.#altText) {
+        this.#altTextButton.title = this.#altText;
+      }
+    }
```

Comment:
About the first point: if the user writes something, save the `done` icon is displayed. If they edit again, remove the text, then they can't save it except in adding something, then it's still `done`, or in setting it as decorative and again it's `done`.

Reply:
> If they edit again, remove the text, then they can't save it except in adding something,

I wonder if that may possibly cause some user confusion, since currently the only way to "remove" a previously saved altText is to change it to decorative?

Although, it might be better to leave this as-is and wait for any bug reports before tweaking this work-flow?

Reply:
So I pinged UX about your point and the answer is "Very good points, for now we keep the flow as it is implemented  and we'll see later if we want to change it".